src/connectors/connector.ts

```diff
@@ -44,8 +43,8 @@ export abstract class Connector extends EventEmitter<ConnectorEvents> {
   abstract connect(): Promise<ConnectorData>
   /** Disconnect wallet. */
   abstract disconnect(): Promise<void>
-  /** Get current account. */
-  abstract account(): Promise<AccountInterface>
+  /** Get current account silently. Return null if the account is not authorized */
+  abstract account(): Promise<string | null>
```

Comment:
is still compatible with starknet-react by changing this?

Reply:
We will have to create a PR on starknet-react with new changes. starknet-react doesn't support the new interface. Should I add Francesco to this review?

Reply:
any news about this, how we want to handle?

Reply:
Francesco will have time next week to check too